xml/s4s_installation_sap.xml

```diff
@@ -961,4 +961,151 @@ SAP-NetWeaver-740-SR2-Installation-Export-CD-3-3</screen>
+                     #                                                                                                                                                                         #
+                     # Installation service 'SAP NetWeaver 7.5 > MaxDB > SAP Systems > Application Server ABAP > Distributed System > ASCS Instance', product id 'NW_ABAP_ASCS:NW750.ADA.ABAP' #
+                     #                                                                                                                                                                         #
+                     ###########################################################################################################################################################################
+
+#USED PARAMETERS ##masterPwd## ##sid## ##kernel## ##ascsVirtualHostname## ##instanceNumber## ##scsVirtualHostname##
+
+# Password for the Diagnostics Agent specific <dasid>adm user. Provided value may be encoded.
+DiagnosticsAgent.dasidAdmPassword =
+
+# Windows domain in which the Diagnostics Agent users must be created.
+# The property is Microsoft Windows only. This is an optional property.
+DiagnosticsAgent.domain =
+
+# Password for the Diagnostics Agent specific SAPService<DASID> user. Provided value may be encoded.
+# The property is Microsoft Windows only.
+DiagnosticsAgent.sapServiceDASIDPassword =
+
+NW_GetMasterPassword.masterPwd = ##masterPwd##
+
+# Human readable form of the Default Login language - used for instant in the preselection in SAPGUI. This Parameter is potentialy asked in addition in the dialog that also asks for the SID. It is not asked in all this dialogs, but only in systems that have ( potentialy beside others ) an ABAP stack. It is ask for installation but not for system copy. It is asked in those installations, that perform the ABAP load. That could be the Database Load installation in case of a distributed szenario, or in the Standard Installer, that is not distributed at all. This Parameter is saved in the Default Profile. It is has no influence on Language settings in a Java Stack. Valid names are stored in a table of the subcomponent NW_languagesInLoadChecks. The available languages must be declaired in the LANGUAGES_IN_LOAD parameter of the product.xml . In this file the one character representation of the languages is uses. Check the same table in the subcomponent mentioned above.
+NW_GetSidNoProfiles.SAP_GUI_DEFAULT_LANGUAGE =
+
+# The drive to use (windows only)
+NW_GetSidNoProfiles.sapdrive =
+
+# The /sapmnt path (unix only)
+NW_GetSidNoProfiles.sapmnt = /sapmnt
+
+# The SAP System ID of the system to install
+NW_GetSidNoProfiles.sid = ##sid##
+
+# Will this system be unicode system?
+NW_GetSidNoProfiles.unicode = true
+
+NW_SAPCrypto.SAPCryptoFile = /data/SAP_CDs/745-UKERNEL-SAP-Unicode-Kernel-745/DBINDEP/SAPEXE.SAR
```

Comment:
Is it wise to keep this path in the docs? -- I suspect that if you use the same version and copy the files with YaST, this path is always the same. However, that is quite a lot of assumptions -- should we use a replacable here?